lib/worker/src/db/publications.rs

```diff
@@ -57,6 +58,58 @@ pub async fn pub_table_insert(pool: &PgPool, stmts: Vec<String>) -> Result<()> {
     Ok(txn.commit().await?)
 }
 
+// Lists cids of a given publication
+pub async fn pub_cids(
+    pool: &PgPool,
+    ns: String,
+    rel: String,
+    limit: i32,
+    offset: i32,
+) -> Result<Vec<String>> {
+    let res = sqlx::query(
+        "SELECT cid FROM jobs 
+                                                JOIN namespaces ON namespaces.id = jobs.ns_id 
+                                                WHERE name=$1 AND relation = $2
+                                                ORDER BY jobs.id DESC
+                                                LIMIT $3
+                                                OFFSET $4",
+    )
+    .bind(ns)
+    .bind(rel)
+    .bind(limit)
+    .bind(offset)
+    .fetch_all(pool)
+    .await?;
+
+    let cids = res
+        .iter()
+        .map(|row| multibase::encode::<Vec<u8>>(Base::Base32Lower, row.get("cid")))
+        .collect();
+
+    Ok(cids)
+}
+
+// Insert a job (used for testing)
+pub async fn pub_jobs_insert(
```

Comment:
since this function is quite independent and used only for testing, maybe we could add it in `integration.rs` itself?